### PR TITLE
chore: add all resources individually to tarball

### PR DESCRIPTION
**Pull request by toddbaert**
Helm doesn't automatically perform CRD upgrades. One thing we can do to help this is publish them separately so admins can upgrade these manually apart from helm.

This PR helps the situation by also publishing all our resources individually and including them in the helm tarball, so admins can easily update the CRDs with something like:

```helm template open-feature-operator-v0.2.28.tgz -s templates/apiextensions.k8s.io_v1_customresourcedefinition_featureflagconfigurations.core.openfeature.dev.yaml | kubectl apply -f -```

Besides this added benefit, there's no change. You can still install the operator with `helm install/upgrade` as before.

### CODE REVIEW

File: chart/open-feature-operator/templates/apiextensions.k8s.io_v1_customresourcedefinition_featureflagconfigurations.core.openfeature.dev.yaml
Comment (toddbaert) [inline chart/open-feature-operator/templates/apiextensions.k8s.io_v1_customresourcedefinition_featureflagconfigurations.core.openfeature.dev.yaml:556]: Github is seeing this as a move, but it's actually just getting rid of the old "all-in-one" rendered.yaml.

**Review by beeme1mr — APPROVED** [review]
Looks like a good change to me, thanks!

**Review by skyerus — APPROVED** [review]

**Comment by beeme1mr** [comment]
@toddbaert can you please address the merge conflicts? Once this is merged, please create a separate PR to update the docs.

**Comment by toddbaert** [comment]
> @toddbaert can you please address the merge conflicts? Once this is merged, please create a separate PR to update the docs.

I'd like to wait for a release to update the docs, or it won't be possible/accurate. I'll update the docs then.

**Pull request merged**
